### DISCUSSION TITLE
RT Dynamic Shadows

### DISCUSSION THREAD

**Discussion by ak-988** [Novum/vkQuake#668] [Q&A] [answered]
Maybe I am oblivious but I don't see a difference enabling "Dynamic Shadows" in the menu or "r_rtshadows" in the console with the newest vkQuake release. I usually use rerelease content but original id1 is the same in case that matters. Is there a good way to test if it's working? I'm on Windows 10, 3080ti with 528.49 driver.

**Comment by vide0hanz** [comment, marked as answer]
Its a subtle change. Assuming `r_dynamic` and `r_rtshadows` both = 1 you can notice a subtle shadow effect when a dynamic light interacts with most objects/geometry. It can probably be easiest to observe by giving yourself quad damage (`impulse 255`) and walking near health and ammo pickups. It can also be observed pretty easily by firing a rocket near objects, such as the spiked gemoetry in the HARD skill hallway of the start menu. 

Another area where this feature is more noticable is in the start map. Start a new game and stand still, observe the lava ball that shoots up in the HARD hallway. Try setting `r_rtshadows 0` and notice how the dynamic light from the lava ball can be seen in the NORMAL hallway.

**Comment by temx** [comment]
Some examples:
![](https://user-images.githubusercontent.com/24851409/225409224-6ab5e690-3707-456c-a0ad-2dc9eb52f89b.gif)

https://user-images.githubusercontent.com/24851409/225409305-a79ff732-3342-4fc9-b2ea-6729ecabf1b0.mp4

https://user-images.githubusercontent.com/24851409/225409319-5be25444-46d5-4cd1-9d70-c055a9f93c90.mp4

**Reply by vide0hanz** [reply]
Beat me to it, lol.

**Comment by ak-988** [comment]
That's exactly what I wanted to know, thank you very much for the info and examples! Seems everything is working nicely. I appreciate the subdued tasteful effect a lot. Must say vkQuake looks fantastic with this and the rerelease content which is now dead simple to use. Absolutely smooth and responsive too. Really looking forward to spending more time with it. Thanks for everyone's work on it :)

**Comment by Novum** [comment]
This is also expanded to enemies now in latest code so it's much more impactful
